### ISSUE TITLE
If I don't have training data that uses the default parameters, how can I enhance my speech with a custom model?

### ISSUE THREAD

**Issue by WHUynl** [manhdung20112000/speech-enhancement#7]
Dear,Friends. 
  this is your original config.py.
![image](https://user-images.githubusercontent.com/43106285/160350715-b7a4c783-18f4-4f92-877b-0024f154f6d4.png)
And this is mine
![image](https://user-images.githubusercontent.com/43106285/160351007-209fa3fb-b6a2-43b1-8a0b-0598702467de.png)
And I use my final .pth and the result is awful.
I don't know which files need to be modified and how to modify those files.
PS: I also changed the net-structure,and I don't know  if this has an impact（But there were no problems with the whole training process）
Hope you can solve my problems,thank you very much!

**Comment by nmd2k** [comment]
Hello there, sorry for my late reply 👋
In idea environment, you might only have to tune the hyperparameter like learning rate, number of epoch, drop rate, input size or even optimizer (I believe it's stored in file `config.py`, btw I wouldn't recommend you to follow this style of code, try `argparse`).
However, it's possible that my model is not working as expect. Tuning or customizing U-net is not a bad idea in the end of the day.

Have a beautiful day my friend, hope you've overcame this issue by the time I reply it LOL 😁
